<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>451</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>301</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string>             输入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>   输入图像</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐播放控制</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>111</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="QPushButton" name="last_song_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>上一首</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_song_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>下一首</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <widget class="QPushButton" name="ManualButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>手动控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="VisualButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>视觉控制</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="PlayBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
